scripts/linux/publishReleasePackages.sh

```diff
@@ -239,7 +235,11 @@ publish_to_github()
 
         #Create Release Page
         url="https://api.github.com/repos/Azure/azure-iotedge/releases"
-        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" '{tag_name: $version, name: $version, target_commitish:"main", body: $body}')
+        reqBody='{tag_name: $version, name: $version, target_commitish:"main", draft: true, body: $body}'
+        if [[ $SKIP_UPLOAD == "false" ]]; then
+            reqBody='{tag_name: $version, name: $version, target_commitish:"main", body: $body}'
+        fi
```

Comment:
The first time edgelet build pipeline call this script to create a draft page, it doesn't upload any artifacts. For that run we want to create a draft page. Then later when the artifacts are ready and the version files are updated, we want to push the official release page and tag the commit. The tagging is done from the pipeline along with the file version updated.

Reply:
I'm getting confused by what is going on here I think. Questions:
1. Edgelet release build pipeline ends up calling this function once, or multiple times?
2. Is `SKIP_UPLOAD` named like this because when we upload the packages we don't want the release to be a draft anymore?

Reply:
> 1. Edgelet release build pipeline ends up calling this function once, or multiple times?

The script is called twice! First time to setup the release page with all the changelog description in a draft mode (without uploading the artifacts).
https://github.com/Azure/iotedge/blob/release/1.2/builds/misc/packages-release.yaml#L237
 Second time when the artifacts are done building and need to be uploaded to the github. 
 https://github.com/Azure/iotedge/blob/release/1.2/builds/misc/packages-release.yaml#L359

> 2. Is SKIP_UPLOAD named like this because when we upload the packages we don't want the release to be a draft anymore?

Per (1), the `SKIP_UPLOAD` was utilized during the first time the script was called since there was no artifact to be uploaded.